config.yaml

```diff
@@ -1258,6 +1270,122 @@ languages:
           weight: 165
           parent: customnav_agentnav_basic_usage
 
+        ## TRACING/APM - Setup basic
+        - identifier: customnav_tracingnav_setup
+          name: "APM"
+          url: "agent/apm/"
+          weight: 13
+          parent: menu_references_agent
+
+        - identifier: customnav_tracingnav_java
+          name: "Java"
+          url: "agent/apm/java/"
+          weight: 1520
+          parent: customnav_tracingnav_setup
+        - identifier: customnav_tracingnav_python
+          name: "Python"
+          url: "agent/apm/python/"
+          weight: 1530
+          parent: customnav_tracingnav_setup
+        - identifier: customnav_tracingnav_ruby
+          name: "Ruby"
+          url: "agent/apm/ruby/"
+          weight: 1540
+          parent: customnav_tracingnav_setup
+        - identifier: customnav_tracingnav_go
+          name: "Go"
+          url: "agent/apm/go/"
+          weight: 1550
+          parent: customnav_tracingnav_setup
+        - identifier: customnav_tracingnav_javascript
+          name: "Node.js"
+          url: "agent/apm/nodejs/"
+          weight: 1560
+          parent: customnav_tracingnav_setup
+        - identifier: customnav_tracingnav_dotnet
+          name: ".NET"
+          url: "agent/apm/dotnet/"
+          weight: 1570
+          parent: customnav_tracingnav_setup
+        - identifier: customnav_tracingnav_php
+          name: "PHP"
+          url: "agent/apm/php/"
+          weight: 1580
+          parent: customnav_tracingnav_setup
+        - identifier: docker_ecs
+          name: "Tracing ECS application"
```

Comment:
Also I dont think "ECS application" is the right way to phrase this.  The application is being run in a a container orchestrated on ECS, but its not an application specific to ECS.

Per Amazon guidelines:
Amazon Elastic Container Service
or
Amazon ECS

Reply:
I just moved the content for now to remove the other PR, I think this content could go the Datadog - Amazon Elastic Container Service integration page directly after all.